apps/storybook/src/TiledHeatmapMesh.stories.tsx

```diff
@@ -27,46 +28,49 @@ import type { Vector2 } from 'three';
 import FillHeight from './decorators/FillHeight';
 
 // See https://en.wikipedia.org/wiki/Mandelbrot_set
-function mandelbrot(
-  iterations: number,
-  xDomain: Domain,
-  yDomain: Domain,
-  size: Size
-): NdArray<Float32Array> {
-  const { width, height } = size;
-  const xRange = xDomain[1] - xDomain[0];
-  const yRange = yDomain[1] - yDomain[0];
+const mandelbrot = greenlet(
+  async (
+    iterations: number,
+    xDomain: Domain,
+    yDomain: Domain,
+    size: Size
+  ): Promise<Float32Array> => {
+    const { width, height } = size;
+    const xRange = xDomain[1] - xDomain[0];
+    const yRange = yDomain[1] - yDomain[0];
 
-  const array = ndarray(new Float32Array(height * width), [height, width]);
+    const array = new Float32Array(height * width);
```

Comment:
I can't use `ndarray` inside the worker because of the way `greenlet` works. It's limited to functions that don't rely at all on external scope.